src/stylesheets/home.scss

```diff
@@ -1,13 +1,22 @@
 @import "colors";
-
-#h {
-  width: 100%;
-  margin:0px;
-  padding: 0px;
-}
+@import "screens";
 
 #home {
-  min-height: calc(100vh - 55px);
+  background-image: url('https://static.pexels.com/photos/24746/pexels-photo-24746.jpg');
```

Comment:
Agreed, we have #129 for this but I don't have an alternative! @Hitenp1 found a nice bright one the other day